### PR TITLE
Create TouTiao.xml

### PR DESCRIPTION
**Pull request by renyouguo**

### CODE REVIEW

**Comment by renyouguo** [comment]
@Hainish

**Comment by Hainish** [comment]
Travis tests are failing here
https://travis-ci.org/EFForg/https-everywhere/jobs/153571951

File: src/chrome/content/rules/TouTiao.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		m.pstatp.com
+		s[4-6].pstatp.com
+		(p|v)[0-10].statp.com
+
+-->
+
+<ruleset name="toutiao.com (partial)">
+
+	<target host="toutiao.com" />
+	<target host="app.toutiao.com" />
```
Comment (Hainish) [inline src/chrome/content/rules/TouTiao.xml]: It looks like there are significant differences visiting this over `http` vs `https`

File: src/chrome/content/rules/TouTiao.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	<target host="m.toutiao.com" />
```
Comment (Hainish) [inline src/chrome/content/rules/TouTiao.xml]: I see differences here too

**Comment by Hainish** [comment]
@renyouguo submitted PRs should have roughly the same functionality and look on the `https` site as the `http` site.  It looks like the `https` endpoints in multiple hosts for this PR look or act differently.

**Comment by jeremyn** [comment]
I closed pull request #4806 as a duplicate of this pull request, but for completeness I'm noting here that the changes in #4806 included a `securecookie` instruction while the changes merged in this pull request did not.

**Comment by renyouguo** [comment]
@jeremyn I do not know how to check the securecookie.

**Comment by jeremyn** [comment]
@renyouguo I use the Firefox [Storage Inspector](https://developer.mozilla.org/en-US/docs/Tools/Storage_Inspector), `Cookies > isSecure`. Note this doesn't seem to work in private browsing mode.

**Pull request merged**